Mathlib/Data/List/Rotate.lean

```diff
@@ -374,7 +374,7 @@ def IsRotated : Prop :=
   ∃ n, l.rotate n = l'
 
 @[inherit_doc List.IsRotated]
-infixr:1000 " ~r " => IsRotated
+infixr:50 " ~r " => IsRotated
```

Comment:
Does this match the precedence of `Perm`? If so, can you add a `--` comment to that effect?

bors d+

Reply:
```suggestion
-- This matches the precedence of the infix `~` for `List.Perm`, and more generally of relation infixes
infixr:50 " ~r " => IsRotated
```